yoga/model/__init__.py

```diff
@@ -1,11 +1,13 @@
 from .assimp import (assimp_import_from_bytes, assimp_export_to_bytes)
 from .options import (normalize_options, extract_image_options)
+from ..options import normalize_options as normalize_global_options
 from .helpers import model_embed_images
 
 import os.path
 
 
 def optimize(input_file, output_file, options={}):
```

Comment:
quiet ad verbose are not realy options of the optimize function as it does not change the output, maybe can be better to add them this way:

```python
def optimize(input_file, output_file, options={}, quiet=False, verbose=False):
```